extension/src/experiments/model/collect.ts

```diff
@@ -86,7 +86,7 @@ const getDisplayNameOrParent = (
     return `(${shortenForLabel(checkpointParent)})`
   }
   if (name) {
-    return `[${name}]`
```

Comment:
Did we want to take off brackets in all secondary names (quick pick, tree, table) or just take off the brackets in the table? Currently this pr takes off them in all places but we could easily just do the table if preferred :)